lib/themes/dosomething/paraneue_dosomething/js/components/Slideshow.js

```diff
@@ -20,9 +21,16 @@ class Slideshow extends React.Component {
   }
 
   render() {
+    const user = this.props.user;
+    const campaign = this.props.campaign;
+
+    const slides = this.props.slides.map((slide, index) => {
+      return React.createElement(slide, {key: index, campaign: campaign, user: user});
+    });
+
     return (
       <div className="slideshow">
-        {this.props.slides[this.state.activeSlide]}
+        {slides}
```

Comment:
I'm not sure how I feel about this. The reason I had it the previous way was, if you're on slide 1, you only need to render slide 1. With this it seems like we would rerender the entire slideshow every state change, which seems very expensive with no real upside to it.